PyRDF/CallableGenerator.py

```diff
@@ -21,6 +21,45 @@ def __init__(self, head_node):
         """
         self.head_node = head_node
 
+    def get_action_nodes(self, node_py=None):
+        """
+        Recurses through PyRDF graph and collects
+        the PyRDF nodes.
+
+        Parameters
+        ----------
+        node_py (optional)
+            The current state's PyRDF node. If `None`,
+            it takes the value of `self.head_node`.
+
+        Returns
+        -------
+            list
+                A list of action nodes in the same order as the
+                order of execution in the callable function.
```

Comment:
Can we say: A list of the action nodes of the graph in DFS order, which coincides with the order of execution in the callable function.

Reply:
Yup @etejedor . That sounds better. I'll modify it 👍